addons/account/models/account_move_send.py

```diff
@@ -421,6 +421,7 @@ def _send_mail(self, move, mail_template, **kwargs):
             .with_context(
                 no_new_invoice=True,
                 mail_notify_author=author_id in partner_ids,
+                email_show_footer=True,
```

Comment:
Could be great to prepare real method parameters propagation instead of relying on context completely. We cannot modify `_notify_by_email_prepare_rendering_context` as it would break overrides. However message_post receives kwargs that are propagated to `_notify_thread_by_email`. We could enable new rendering parameters that are already added in `_get_notify_valid_parameters` (whitelist of parameters). Then in _notify_thread_by_email we have to fallback on context, and at that point we put them in context (update context for the sub calls with parameters received in context). We could allow 'force_email_footer' and 'force_email_header' for example.

Avantage: when reaching master we already have part of the "clean implementation" job done, and we would just have to add those parameters in the sub methods of _notify_thread_by_email. Parameters would already be explicit in part of the code, making them a bit less obscure than pure context keys. What do you think ?